operations/utility_storage/Manage_Ceph_Services.md

```diff
@@ -228,7 +228,7 @@ Two SSH sessions are required. One to do the work from and another that is runni
      mgr                            3/3  9s ago     4d   ncn-s001;ncn-s002;ncn-s003;count:3                             registry.local/ceph/ceph:v15.2.8  5553b0cb212c
      mon                            3/3  9s ago     4d   ncn-s001;ncn-s002;ncn-s003;count:3                             registry.local/ceph/ceph:v15.2.8  5553b0cb212c
      osd.all-available-devices      6/6  9s ago     4d    *                                                              registry.local/ceph/ceph:v15.2.8   5553b0cb212c
-     rgw.site1.zone1                3/3  9s ago     4d   ncn-s001;ncn-s002;ncn-s003;ncn-s004;ncn-s005;ncn-s006;count:3  registry.local/ceph/ceph:v15.2.8  5553b0cb212c
+     rgw.site1                3/3  9s ago     4d   ncn-s001;ncn-s002;ncn-s003;ncn-s004;ncn-s005;ncn-s006;count:3  registry.local/ceph/ceph:v15.2.8  5553b0cb212c
```

Comment:
add some spaces between  rgw.site1 and 3/3 so the columns line up - check others files too as I see this in other places where .zone1 was removed from the command output

Reply:
Also, feel free to run these commands on Beau and then you can copy and paste the output from Beau into the files.